src/python/pants/backend/python/target_types_rules.py

```diff
@@ -187,7 +186,7 @@ async def generate_targets_from_python_test_utils(
 # -----------------------------------------------------------------------------------------------
 
 
-@rule(desc="Determining the entry point for a `pex_binary` target", level=LogLevel.DEBUG)
+@rule(desc="Determining the entry point for a `pex_binary` target")
```

Comment:
Why? This won't show up in the Dynamic UI anymore iirc

Reply:
Because in traces at DEBUG level, all of its instances took < 20ms, which meant that if it did show up, it would be for too short a time for it to be meaningful.

Reply:
Okay, so then `desc` should probably be removed too, right?

Reply:
The description is harmless I think? It mostly just acts as documentation for the `@rule`, and if you were to run at `TRACE` level you'd get a human readable description.

Most rules at `DEBUG` should probably have descriptions, but there is no harm in `@rule`s at `TRACE` having descriptions.